source/draw/DrawState.ooc

```diff
@@ -17,12 +17,18 @@ import Mesh
 
 // See README.md about input arguments and coordinate systems
 
+BlendMode: enum {
+	Fill
+	Add
```

Comment:
Defined as `glBlendFunc(GL_ONE, GL_ONE_MINUS_SRC_COLOR)`
The source color is additive but brighter source also reduces destination intensity which makes edges softer.

One could also define `glBlendFunc(GL_ONE, GL_ONE)` for a pure additive blending.